README.md

```diff
@@ -1,69 +1,113 @@
-# usecases
+# Storyboards
 
-## Build Setup
+This repository contains the source code for the
+[EUCP](https://www.eucp-project.eu/) [Storyboard
+website](https://eucp-project.github.io/storyboards/).
 
-```bash
-# install dependencies
-$ npm install
-
-# serve with hot reload at localhost:3000
-$ npm run dev
+The website has been built with [Nuxt](https://nuxtjs.org), using
+[nuxt-content](https://content.nuxtjs.org/) for authoring stories and
+[tailwindcss](https://tailwindcss.com/docs/installation) + [tailwind
+typography](https://tailwindcss.com/docs/typography-plugin) for styling. It is
+hosted on [GitHub pages](https://nuxtjs.org/deployments/github-pages/).
 
-# build for production and launch server
-$ npm run build
-$ npm run start
-
-# generate static project
-$ npm run generate
-```
-
-For detailed explanation on how things work, check out the [documentation](https://nuxtjs.org).
+## Writing a story
 
-## Special Directories
+All stories are stored in the `static/stories` folder. Each story consists of a
+markdown file and a folder with some assets (images, etc.) belonging to the
+story. In addition to standard markdown, the `:::Chapter{}` directive is used to
+break the story into parts that can be displayed individually.
 
-You can create the following extra directories, some of which have special behaviors. Only `pages` is required; you can delete them if you don't want to use their functionality.
+An example story might look like this:
```

Comment:
The example looks awesome. It could be even more "user friendly" by adding an image (snapshot) below showing how it looks like with the given script, which is less horrible for domain scientist with little coding experience.

Reply:
Thanks @geek-yang, I have added this